### DISCUSSION TITLE
Why can't Shift and Trace be used in the upper and lower layers

### DISCUSSION THREAD

**Discussion by hhsyya** [opentoonz/opentoonz#6528] [Q&A]
### Ask a question related to OpenToonz

When painting on the upper layer, it is not possible to write through to the Shift and Trace of the lower layer at will. it would be great if i could

![2023-07-19 10-10-33-87](https://github.com/opentoonz/opentoonz/assets/103864209/10ee2db8-60eb-43b9-a420-1c5193d37ad5)

**Comment by RodneyBaker** [comment]
No one has offered any advice here so perhaps we can assume no one has seen the request for information.

Will transfer this to Discussions where Shift and Trace can be more fully explored and any issues identified and recommended for development.
